### ISSUE TITLE
rbenv_prefix overwritten by commit that introduced `set :bundle_bins,`

### ISSUE THREAD

**Issue by pjammer** [seuros/capistrano-puma#138]
`set :bundle_bins, fetch(:bundle_bins).to_a.concat(%w{ puma pumactl })` and resulting changes to the `puma.rake` file in the commit https://github.com/seuros/capistrano-puma/commit/2a0c2dc3b2dc7e75ccc8640b4595e5afb4f3a10d appear to have broken any puma restart (in my case).

i cowboy coded locally by commenting out the changes in this commit back to using `execute :bundle, 'exec', :pumactl, "-S #{fetch(:puma_state)} -F #{fetch(:puma_conf)} #{command}"` for instance.

I'm not proficient enough in capistrano to understand where execute gets it's call for bundle from, but in a code base i inherited they used a `rbenv_prefix` and that basically hardcodes the rbenv variables and paths it needs to run a 'normal' `bundle exec`

here is a sample: `set :rbenv_prefix, "RBENV_ROOT=#{fetch(:rbenv_path)} RBENV_VERSION=#{fetch(:rbenv_ruby)} #{fetch(:rbenv_path)}/bin/rbenv exec"`

without this, and using the existing gem that way it is, you get an error similar to #131 had.

```bash 
DEBUG [b57776c7] 	bash: bundle: command not found
(Backtrace restricted to imported tasks)
cap aborted!
SSHKit::Runner::ExecuteError: Exception while executing as deploy@blah.com: pumactl exit status: 127
pumactl stdout: Nothing written
pumactl stderr: bash: bundle: command not found

SSHKit::Command::Failed: pumactl exit status: 127
pumactl stdout: Nothing written
pumactl stderr: bash: bundle: command not found

Tasks: TOP => puma:phased-restart
(See full trace by running task with --trace)
The deploy has failed with an error: Exception while executing as deploy@blah.com: pumactl exit status: 127
pumactl stdout: Nothing written
pumactl stderr: bash: bundle: command not found

```

Ideally I'd make the new change work with this prefix, but how execute gets inherited is beyond me right now. Just thought I'd leave this here for now and see if it's something we did wrong, or if we need to add the `rbenv_prefix` somehow to the `puma.rake` file. dunno. spitballing.

**Comment by seuros** [comment]
You should install `bundler` in your servers.

**Comment by pjammer** [comment]
Bundler is on the servers. all the other parts of capistrano are running and then the restart happens and this is when it broke.

**Comment by manme** [comment]
have the same problem with capistrano-puma
resolved with using v1.2.1
